homeassistant/components/mvglive/sensor.py

```diff
@@ -51,6 +52,7 @@
                     CONF_PRODUCTS, default=DEFAULT_PRODUCT
                 ): cv.ensure_list_csv,
                 vol.Optional(CONF_TIMEOFFSET, default=0): cv.positive_int,
+                vol.Optional(CONF_ENTRIES, default=10): cv.positive_int,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
This little fix changes nothing for existing users, it only adds the possibility to specify it.
Doing a full refactoring of an integration, I did not even write by myself is a big obstacle. 
The alternative is that integration stays as it is and gets no improvement. I think this is the worse option.

Reply:
Ok, I'll close here then. Thanks for your contribution. :+1:

Reply:
Your decision. As i said, leaving it like this would be worse, than adding a non breaking change and not comply with the guidelines, which it already does.
I fully understand, that you don't want new integrations with yaml config, but fixing an existing one?

Reply:
Yes, we have decided to not allow any changes to platform schemas of existing concerned integrations according to ADR 10.

> Changes to existing YAML configuration for these same existing integrations, will no longer be accepted.